client/state/action-types.js

```diff
@@ -61,4 +61,8 @@ export const SUPPORT_USER_TOKEN_FETCH = 'SUPPORT_USER_TOKEN_FETCH';
 export const SUPPORT_USER_TOKEN_SET = 'SUPPORT_USER_TOKEN_SET';
 export const SUPPORT_USER_RESTORE = 'SUPPORT_USER_RESTORE';
 export const USER_RECEIVE = 'USER_RECEIVE';
+export const VIDEO_FETCH = 'VIDEO_FETCH';
```

Comment:
In the [action-types.js](https://github.com/Automattic/wp-calypso/blob/master/client/state/action-types.js) there are examples for both `FETCH` and `REQUEST`. My thinking was that I am *fetching* existing data (a resource, like e.g. settings) but I am *requesting* an operation (like those, arguably not very RESTful actions that introduce a state change for a resource, like activate, star, or publish).
I do not feel the need to have this distinction in the language, and for me `request` covers both scenarios. I only used `fetch` because the existing code seemed to have this distinction. I am happy to agree on the term `request` and would prefer using that in my action names.